Update readme.md (#88)
Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io: docs/partners/2-certification/certification-for-data-providers/readme.md

Specific certifications describe additional requirements; those can be found
in [Available Certifications](../available-certifications/readme.md).

## Alignment with new Break-Rest Development Cadence

> ![Break-Rest Development Cadence Table](https://edfi.atlassian.net/wiki/download/attachments/20876154/image-2024-3-30_10-49-43.png?version=1&modificationDate=1711813783893&cacheVersion=1&api=v2 "Break Rest Development Cadence")

Reviewer: ```suggestion
![Break-Rest Development Cadence Table](https://edfidocs.blob.core.windows.net/$web/img/reference/release-cadence.png "Break Rest Development Cadence")
```

Certification requirements are aligned to the Ed-Fi Data and API Standard Release Cadence

> * Support goes from 3 to 4 years

Reviewer: Extra line break above the bullets makes this more readable in plain text.

```suggestion

* Support goes from 3 to 4 years
```

Should there be a period at the end of each bullet point?

Author: I'll follow the practice you've established already

* Certification will continue to be annual 
* Vendors should certify on everything that is active, we always encourage on the latest 
* There will be some experimental areas of domains to change more rapidly than the cadence (those areas will not be included in testing)

> Visit this section for more discussion on the [Ed-Fi Data And API Standard Release Cadence](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/20876154). 

Reviewer: ```suggestion
Visit this section for more discussion on the [Ed-Fi Data And API Standard Release Cadence](/reference/roadmap/cadence). 
```
